eitprocessing/binreader/sequence.py

```diff
@@ -346,7 +353,7 @@ def select_by_time(
 
         return self.select_by_indices(slice(start_index, end_index))
 
-    __getitem__ = select_by_indices
+    __getitem__ = select_by_indices  # TODO: why not immediately define it??
```

Comment:
When writing this, I was still debating whether `__getitem__` would have to map to `select_by_indices` or `select_by_time`.

For completeness, `select_by_indices` could be available separately, similar to how e.g. pandas handles slicing, but this is not required.